core/cmd/client.go

```diff
@@ -160,7 +161,20 @@ func (n ChainlinkAppFactory) NewApplication(cfg config.GeneralConfig, db *sqlx.D
 		KeyStore:         keyStore.Eth(),
 		EventBroadcaster: eventBroadcaster,
 	}
-	chainSet, err := evm.LoadChainSet(ccOpts)
+	var chainSets chainlink.Chains
+	chainSets.EVM, err = evm.LoadChainSet(ccOpts)
+	if err != nil {
+		appLggr.Fatal(err)
+	}
+	terraLggr := appLggr.Named("Terra")
```

Comment:
Do we usually specify the name inside or outside the constructor?

Reply:
Generally the constructors add their own name so that every caller doesn't have to.